sdk/monitor/azure-monitor-opentelemetry/setup.py

```diff
@@ -88,14 +88,15 @@
         "azure-core<2.0.0,>=1.28.0",
         "azure-core-tracing-opentelemetry~=1.0.0b11",
         "azure-monitor-opentelemetry-exporter~=1.0.0b26",
-        "opentelemetry-instrumentation-django~=0.42b0",
-        "opentelemetry-instrumentation-fastapi~=0.42b0",
-        "opentelemetry-instrumentation-flask~=0.42b0",
-        "opentelemetry-instrumentation-psycopg2~=0.42b0",
-        "opentelemetry-instrumentation-requests~=0.42b0",
-        "opentelemetry-instrumentation-urllib~=0.42b0",
-        "opentelemetry-instrumentation-urllib3~=0.42b0",
+        "opentelemetry-instrumentation-django~=0.46b0",
+        "opentelemetry-instrumentation-fastapi~=0.46b0",
+        "opentelemetry-instrumentation-flask~=0.46b0",
+        "opentelemetry-instrumentation-psycopg2~=0.46b0",
+        "opentelemetry-instrumentation-requests~=0.46b0",
+        "opentelemetry-instrumentation-urllib~=0.46b0",
+        "opentelemetry-instrumentation-urllib3~=0.46b0",
         "opentelemetry-resource-detector-azure~=0.1.4",
+        "opentelemetry-sdk~=1.25.0",
```

Comment:
Be careful with the versioning. I believe this should be "opentelemetry-sdk~=1.25"

Reply:
Will fix.

Reply:
Done.